### ISSUE TITLE
Meaning of hyperparameter options in tuning

### ISSUE THREAD

**Issue by manmeet3591** [Yura52/tabular-dl-revisiting-models#36]
I am confused with what the values in the toml file for tuning actually mean. For example, d_layers is having 4 numbers (1,8,1,512) as shown below. Shoudnt it be three numbers, i.e. starting number of layers to test, last number of layers to test and maybe the step size. Sorry if this sounds stupid

[optimization.space.model]
d_layers = [ '$mlp_d_layers', 1, 8, 1, 512 ]
dropout = [ '?uniform', 0.0, 0.0, 0.5 ]

[optimization.space.training]
lr = [ 'loguniform', 1e-05, 0.01 ]
weight_decay = [ '?loguniform', 0.0, 1e-06, 0.001 ]
